src/nomad_simulations/schema_packages/outputs.py

```diff
@@ -66,6 +66,8 @@ class Outputs(ArchiveSection):
         """,
     )
 
+
+class ElectronicStructureOutputs(Outputs):
```

Comment:
Maybe redo this change, i remember we agreed that we, for the moment simply put all properties in Outputs

Reply:
@ladinesa I made this change, please check #175 

I also tested this branch with my h5md parser (both mapping annotations and the old way), and had no issues. I suggest that I merge #175 here and then merge this one. Please confirm.

Reply:
ok, i am also adapting the changes to the parsers.